doc/install/mne_python.rst

```diff
@@ -261,16 +287,24 @@ Most users find it convenient to write and run their code in an `Integrated
 Development Environment`_ (IDE). Some popular choices for scientific
 Python development are:
 
-- `Spyder`_ is an IDE developed by and for scientists who use Python. It is
-  included by default when you install anaconda and can be started from a
-  terminal with the command ``spyder``. If you installed MNE-Python into a
-  separate environment from the ``base`` anaconda environment, you can add
-  Spyder to your MNE-Python environment by running ``conda install spyder``
-  while your MNE-Python environment is active. Spyder is free and open-source.
+- `Spyder`_ is a free and open-source IDE developed by and for scientists who
+  use Python. It is included by default in the ``base`` environment when you
+  install Anaconda, and can be started from a terminal with the command
+  ``spyder`` (or on Windows or macOS, launched from the Anaconda Navigator GUI).
+  If you installed MNE-Python into a separate ``mne`` environment (not the
+  ``base`` Anaconda environment), you can set up Spyder to use the ``mne``
+  environment automatically, by opening Spyder and navigating to
+  :samp:`Tools > Preferences > Python Interpreter > Use the following interpreter`.
+  There, paste the output of the following terminal command::
+
+      $ conda activate mne && python -c "import sys; print(sys.executable)"
+
+  It should be something like ``C:\Users\user\anaconda3\envs\mne\python.exe``
+  (Windows) or ``/Users/user/anaconda3/envs/mne/bin/python`` (macOS).
 - `Visual Studio Code`_ (often shortened to "vscode") is a development-focused
   text editor that supports many programming languages in addition to Python,
-  and has a rich ecosystem of packages to extend its capabilities. Installing
-  `Microsoft's Python Extension
+  includes an integrated terminal console, and has a rich ecosystem of packages
+  to extend its capabilities. Installing `Microsoft's Python Extension
   <https://marketplace.visualstudio.com/items?itemName=ms-python.python>`__ is
   enough to get most Python users up and running. Visual Studio Code is free
   and open-source.
```

Comment:
I had the idea to also add the "VS Code" abbreviation, WDYT?

```
- `Visual Studio Code`_ (often shortened to "VS Code" or "vscode") is a
  development-focused text editor that supports many programming languages in
  addition to Python, includes an integrated terminal console, and has a rich
  ecosystem of packages to extend its capabilities. Installing
  `Microsoft's Python Extension
  <https://marketplace.visualstudio.com/items?itemName=ms-python.python>`__ is
  enough to get most Python users up and running. Visual Studio Code is free
  and open-source.
```

Also, one thing I'd love use to mention is the Pylance extension, even though it's still a preview version. It builds on the Python extension and for me, this thing was really a game changer: for the first time since I started using VS Code in Python, I feel that it's actually _working properly_, and is fun to use. It would make using Python so much better, even for newcomers:

Link to the extension:
https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance

Pylance announcement:
https://devblogs.microsoft.com/python/announcing-pylance-fast-feature-rich-language-support-for-python-in-visual-studio-code/#comments

Reply:
It's just sad that VS Code still can't do proper Python indentation, e.g.:

```python
x = [1, 2, 3,
4, 5, 6]
```

Of course there's a dedicated extension, but this basic feature should really work out of the box. Other than that, VS Code is great, and so is PyCharm.

Reply:
> It's just sad that VS Code still can't do proper Python indentation, e.g.:

+1000



> Other than that, VS Code is great, and so is PyCharm.

I still prefer PyCharm Professional to VS Code :) But VS Code is catching up, quick.